starknet_py/net/schemas/rpc.py

```diff
@@ -147,6 +150,17 @@ def make_dataclass(self, data, **kwargs) -> DeployTransaction:
         return DeployTransaction(**data)
 
 
+class L1HandlerTransactionSchema(TransactionSchema):
+    contract_address = Felt(data_key="contract_address", required=True)
+    calldata = fields.List(Felt(), data_key="calldata", required=True)
+    entry_point_selector = Felt(data_key="entry_point_selector", required=True)
+    nonce = Felt(data_key="nonce", load_default=None)
```

Comment:
Should we allow `None` for these transactions? It seems from the spec that they will always have a nonce, though I'm not 100% sure.